src/workingtitle-vcockpits-instruments-navsystems-g3000/html_ui/WorkingTitle/Pages/VCockpit/Instruments/NavSystems/AS3000/AS3000_Common.js

```diff
@@ -68,18 +88,27 @@ class AS3000_MapElement extends MapInstrumentElement {
         this.setSimVarFromStorage(AS3000_MapElement.VARNAME_ROAD_HIGHWAY_RANGE_ROOT, this.instrument.zoomRanges.indexOf(AS3000_MapElement.ROAD_HIGHWAY_RANGE_DEFAULT));
         this.setSimVarFromStorage(AS3000_MapElement.VARNAME_ROAD_TRUNK_RANGE_ROOT, this.instrument.zoomRanges.indexOf(AS3000_MapElement.ROAD_TRUNK_RANGE_DEFAULT));
         this.setSimVarFromStorage(AS3000_MapElement.VARNAME_ROAD_PRIMARY_RANGE_ROOT, this.instrument.zoomRanges.indexOf(AS3000_MapElement.ROAD_PRIMARY_RANGE_DEFAULT));
-
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_CITY_SMALL_RANGE_ROOT, this.instrument.zoomRanges.indexOf(AS3000_MapElement.CITY_SMALL_RANGE_DEFAULT));
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_CITY_MEDIUM_RANGE_ROOT, this.instrument.zoomRanges.indexOf(AS3000_MapElement.CITY_MEDIUM_RANGE_DEFAULT));
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_CITY_LARGE_RANGE_ROOT, this.instrument.zoomRanges.indexOf(AS3000_MapElement.CITY_LARGE_RANGE_DEFAULT));
         
         // "Other" settings
         this.setSimVarFromStorage(AS3000_MapElement.VARNAME_NORTHUP_ACTIVE_ROOT, 0);
         this.setSimVarFromStorage(AS3000_MapElement.VARNAME_NORTHUP_RANGE_ROOT, this.instrument.zoomRanges.indexOf(AS3000_MapElement.NORTHUP_RANGE_DEFAULT));
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_NORTHUP_ACTIVE_ROOT, 0);
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_NORTHUP_RANGE_ROOT, this.instrument.zoomRanges.indexOf(AS3000_MapElement.NORTHUP_RANGE_DEFAULT));
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_TRACK_VECTOR_SHOW_ROOT, 0);
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_TRACK_VECTOR_LOOKAHEAD_ROOT, AS3000_MapElement.TRACK_VECTOR_LOOKAHEAD_VALUES.indexOf(AS3000_MapElement.TRACK_VECTOR_LOOKAHEAD_DEFAULT));
         this.setSimVarFromStorage(AS3000_MapElement.VARNAME_WIND_SHOW_ROOT, 0);
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_FUEL_RING_SHOW_ROOT, 0);
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_FUEL_RING_RESERVE_ROOT, AS3000_MapElement.VARNAME_FUEL_RING_RESERVE_DEFAULT);
+        this.setSimVarFromStorage(AS3000_MapElement.VARNAME_ALTITUDE_INTERCEPT_SHOW_ROOT, 0);
     }
 
     setSimVarFromStorage(_root, _default) {
-        let value = WTDataStore.get(`${this.simVarNameID}.${_root}`, _default)
+        let value = WTDataStore.get(`${this.simVarNameID}.${_root}`, _default);
         let key = _root + this.simVarNameID;
-        SimVar.SetSimVarValue(key, "number", value)
+        SimVar.SetSimVarValue(key, "number", value);
```

Comment:
hah, thanks for fixing my missing semicolons.  Horrible python-programmer habit :)